src/main/java/org/broadinstitute/consent/http/db/ConsentDAO.java

```diff
@@ -170,9 +170,9 @@ Consent findConsentByAssociationAndObjectId(@Bind("associationType") String asso
     @SqlQuery("select consentId, name, useRestriction from consents where valid_restriction = false ")
     List<UseRestrictionDTO> findInvalidRestrictions();
 
-    @SqlUpdate("update consents set updated = (:consentStatus::int)::bit(1) where consentId = :referenceId")
+    @SqlUpdate("update consents set updated = :updated where consentId = :referenceId")
```

Comment:
I should have caught this in https://github.com/DataBiosphere/consent/pull/571 but missed it.